lib/Elastica/Query/Ids.php

```diff
@@ -91,11 +91,7 @@ public function setType($type)
      */
     public function setIds($ids)
     {
-        if (is_array($ids)) {
-            $this->_params['values'] = $ids;
-        } else {
-            $this->_params['values'] = array($ids);
-        }
+        $this->_params['values'] = is_array($ids) ? $ids : [$ids];
```

Comment:
You can just cast $ids to array:

http://sandbox.onlinephpfunctions.com/code/3dcdc5e1f7bd81e72c7da2548f5bb701e9622b70